kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -116,17 +125,33 @@ include::enricher/_jkube_container_env_java_options.adoc[]
 
 include::enricher/_jkube_controller.adoc[]
 
-include::enricher/_jkube_service.adoc[]
+[[jkube-dependency]]
+==== jkube-dependency
+
+include::enricher/_jkube_docker-registry-secret.adoc[]
+
+include::enricher/_jkube_git.adoc[]
 
 [[jkube-image]]
 ==== jkube-image
 
+include::enricher/_jkube_ingress.adoc[]
+
+[[jkube-metadata]]
```

Comment:
This seems to be duplicate, I think it can be removed (it's already in the include file)